### ISSUE TITLE
Change pitch of audio

### ISSUE THREAD

**Issue by prime31** [jarikomppa/soloud#261]
Is it possible to change pitch without affecting the playback speed?

**Comment by Green-Sky** [comment]
Yes, via a filter. Specifically, the FFTFilter would be capable of doing that.
https://sol.gfxile.net/soloud/fftfilter.html
However you need to implement your own variant, as sad in the link.
